src/schema/v2/fields/pagination.ts

```diff
@@ -203,6 +205,8 @@ export const paginationResolver = <T>({
   size: number
   /** Returned from the 'X-TOTAL-COUNT' header */
   totalCount: number
+
+  resolveNode?: GraphQLFieldResolver<any, any> | null
```

Comment:
What does this do? Can we add a comment similar to the rest of the interface props?

Reply:
added.